backend/src/core/services/phone-number.service.ts

```diff
@@ -1,5 +1,7 @@
 import { parsePhoneNumber, CountryCode } from 'libphonenumber-js/max'
 
+const SG_NUMBER_FORMAT = /^(\+?65)?(6|8|9)\d{7}$/
```

Comment:
Why did you choose to include numbers starting with 6?

Reply:
Was thinking about how technically it is possible to sign up for Telegram with a landline using the phone call verification. Though can remove if it's really an edge case.